src/server.js

```diff
@@ -211,6 +211,46 @@ app.get(
   }
 );
 
+function averageGeolocation(coords) {
+  if (coords.length === 1) {
+    return {
+      latitude: coords[0].lat,
+      longitude: coords[0].long,
+    };
+  }
+  let x = 0.0;
+  let y = 0.0;
+  let z = 0.0;
+
+  if (coords.length < 1) {
+    return {};
+  } else {
+    for (let coord of coords) {
+      let latitude = (coord.lat * Math.PI) / 180;
```

Comment:
can these be const? here and below

Reply:
Variables declared as const cannot be re-assigned. Since we're in a loop, the variable has to change as it increments. That's why let is a good use here.

Reply:
ugh, i forgot that scoping is something that doesn't really work in JS haha